packs/equipment-effects/effect-blood-booster.json

```diff
@@ -72,12 +70,40 @@
                                 ]
                             }
                         ],
+                        "value": "Compendium.pf2e.equipment-srd.Item.Blood Booster (Greater)"
+                    }
+                ],
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.EquipmentVariety",
+                "rollOption": "blood-booster"
+            },
+            {
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "5",
+                        "predicate": [
+                            "blood-booster:blood-booster-lesser"
+                        ],
+                        "value": 5
+                    },
+                    {
+                        "label": "10",
+                        "predicate": [
+                            "blood-booster:blood-booster-moderate"
+                        ],
+                        "value": 10
+                    },
+                    {
+                        "label": "20",
+                        "predicate": [
+                            "blood-booster:blood-booster-greater"
+                        ],
```

Comment:
I added placeholder labels here. This Choice Set is predicated by mutually exclusive, necessary choices, so it should never be visible. The labels can be omitted and it works fine, but I'm a bit queasy about doing that.

Reply:
So, why not create a generic, none visible localized label like PF2E.SpecificRule.Equipment.Variety that contains all of the various grades?

Reply:
I don't follow. Why add stuff to the localization file to be translated if it's not going to be seen? I've only added labels here because I'm not sure if the Choice Set working without them is intended.

Reply:
See Champion Dedication.
It uses localized labels that are not visible. I was going off that example.

Reply:
Right, but I assume those were pre-existing labels. For this I'd just be creating new labels for nobody to use.

Reply:
I'll probably drop it anyway though. I'll just go the long route instead haha